admin/ajax.php

```diff
@@ -92,23 +92,35 @@ function wpseo_kill_blocking_files() {
 
 	check_ajax_referer( 'wpseo-blocking-files' );
 
-	$message = 'There were no files to delete.';
+	$message = 'success';
+	$errors  = array();
+
+	// Todo: Use WP_Filesystem, but not so easy to use in AJAX with credentials form still internal.
 	$options = get_option( 'wpseo' );
 	if ( is_array( $options['blocking_files'] ) && $options['blocking_files'] !== array() ) {
-		$message       = 'success';
-		$files_removed = 0;
-		foreach ( $options['blocking_files'] as $k => $file ) {
-			if ( ! @unlink( $file ) ) {
-				$message = __( 'Some files could not be removed. Please remove them via FTP.', 'wordpress-seo' );
+		foreach ( $options['blocking_files'] as $file ) {
+			if ( is_file( $file ) ) {
+				if ( ! @unlink( $file ) ) {
```

Comment:
I would combine this `if` with the one above.

`if ( is_file( $file )  && ! @unlink( $file ) ) {`